smarty/cli_smirky.py

```diff
@@ -0,0 +1,178 @@
+            help="Filename defining atom OR decorators and associated probabilities. These are combined with atom bases in SMIRKS, for example in '[#6X4,#7X3;R2:2]' 'X4' and 'X3' are ORdecorators. (OPTIONAL)")
+
+    parser.add_option('-A', '--atomANDdecors', metavar="DECORATORS",
+            action='store', type='string', dest='atom_AND_decorators',
+            default='odds_files/atom_decorators.smarts',
+            help="Filename defining atom AND decorators and associated probabilities. These are added to the end of an atom's SMIRKS, for example in '[#6X4,#7X3;R2:2]' 'R2' is an AND decorator. (OPTIONAL)")
+
+    parser.add_option('-o', '--bondORbase', metavar="DECORATORS",
+            action='store', type='string', dest='bond_OR_bases',
+            default='odds_files/bond_OR_bases.smarts',
+            help="Filename defining bond OR bases and their associated probabilities. These are OR'd together to describe a bond, for example in '[#6]-,=;@[#6]' '-' and '=' are OR bases. (OPTIONAL)")
+
+    parser.add_option('-a', '--bondANDdecors', metavar="DECORATORS",
+            action="store", type='string', dest='bond_AND_decorators',
+            default='odds_files/bond_AND_decorators.smarts',
+            help="Filename defining bond AND decorators and their associated probabilities. These are AND'd to the end of a bond, for example in '[#6]-,=;@[#7]' '@' is an AND decorator.(OPTIONAL)")
+
+    parser.add_option('-D', '--atomOddsFile', metavar="ODDSFILE",
+            action="store", type="string", dest="atom_odds",
+            default='odds_files/atom_index_odds.smarts',
+            help="Filename defining atom descriptors and probabilities with making changes to that kind of atom. Options for descriptors are integers corresponding to that indexed atom, 'Indexed', 'Unindexed', 'Alpha', 'Beta', 'All'. (OPTIONAL)")
+
+    parser.add_option('-d', '--bondOddsFile', metavar="ODDSFILE",
+            action="store", type="string", dest="bond_odds",
+            default='odds_files/bond_index_odds.smarts',
+            help="Filename defining bond descriptors and probabilities with making changes to that kind of bond. Options for descriptors are integers corresponding to that indexed bond, 'Indexed', 'Unindexed', 'Alpha', 'Beta', 'All'. (OPTIONAL)")
+
+    parser.add_option("-s", "--substitutions", metavar="SUBSTITUTIONS",
+            action="store", type="string", dest='substitutions_filename',
+            default=None,
+            help="Filename defining substitution definitions for SMARTS atom matches. (OPTIONAL).")
+
+    parser.add_option("-f", "--initialtypes", metavar='INITIALTYPES',
+            action="store", type="string", dest='initialtypes_filename',
+            default=None,
+            help="Filename defining initial (first) fragment types as 'SMIRKS    typename'. If this is left blank the initial type will be a generic form of the given fragment, for example '[*:1]~[*:2]' for a bond (OPTIONAL)")
```

Comment:
All of the examples start with empty smirks, which is the default. 

The other option is to start with a set of initial SMIRKS, which have labels/typename which would correspond to the parameter ID's later when we read in a set of SMIRKS in a SMIRFF. 

I updated the wording a little so maybe it is more clear.